baseplate/chronos.py

```diff
@@ -0,0 +1,22 @@
+"""Extensions to the standard library `datetime` module."""
+
+from datetime import datetime
+
+import pytz
+
+
+def datetime_to_epoch_seconds(dt):
```

Comment:
i'm adding a bunch of type annotations to baseplate this week, would you be interested in adding them here? the [metrics module](https://github.com/reddit/baseplate.py/blob/master/baseplate/metrics.py#L313) is fully typed up now if you want an example.